### DISCUSSION TITLE
when i use GNU Parallel  to analyze data, something goes wrong

### DISCUSSION THREAD

**Discussion by TaoBern** [milaboratory/mixcr#1511] [Q&A] [answered]
when i follow the script on the https://mixcr.com/mixcr/guides/generic-multiplex-bcr/
 For the one sample analysis, it's okay. Successfully aligned reads: 5053118 (95.13%)

The script i used as below:
mixcr analyze generic-bcr-amplicon \
    --species mmu \
    --rna \
    --rigid-left-alignment-boundary \
    --rigid-right-alignment-boundary \
    --tag-pattern '^N{4}ccctcctttaattcccN{22}(R1:*)\^N{4}gaggagagagagagagN{26}(R2:*)' \ 
    --assemble-clonotypes-by {CDR1Begin:FR4End} \
    raw/M1_4T1_replica1_BM_R1.fastq.gz \
    raw/M1_4T1_replica1_BM_R2.fastq.gz \
    results/M1_4T1_replica1_BM

but when I run the samples together using the suggested script, the results look weird.
The script is below 
module load parallel/20210622-GCCcore-10.3.0
ls  raw1/*R1* | \
    parallel -j 4 \
    'mixcr analyze generic-bcr-amplicon -f \
    --species mmu \
    --rna \
    --rigid-left-alignment-boundary \
    --rigid-right-alignment-boundary \
    --tag-pattern "^N{4}ccctcctttaattcccN{22}(R1:*)\^N{4}gaggagagagagagagN{26}(R2:*)" \
    --assemble-clonotypes-by {CDR1Begin:FR4End} \
    {} \
    {=s:R1:R2:=} \
    {=s:.*/:results1/:;s:_R.*::=}' 

But the Successfully aligned reads are all 0%. But the sample under the analysis of the first script is good. It's really weird. I will really appreaciate it if someone can help me find out the reason!

**Comment by mizraelson** [comment, marked as answer]
The command should be:

```
ls raw1/*R1* |
parallel -j 4 \
'mixcr analyze generic-bcr-amplicon -f \
--species mmu \
--rna \
--rigid-left-alignment-boundary \
--rigid-right-alignment-boundary \
--tag-pattern "^N{"4"}ccctcctttaattcccN{"22"}(R1:*)\^N{"4"}gaggagagagagagagN{"26"}(R2:*)" \
--assemble-clonotypes-by {CDR1Begin:FR4End} \
{} \
{=s:R1:R2:=} \
{=s:./:result1/:;s:_R.*::=}'
```

Did you use double quotes inside `{}` in the pattern : `{"4"}` etc.
